.github/workflows/pr-validate-changesets.yaml

```diff
@@ -112,6 +112,18 @@ jobs:
             fi
           fi
 
+          # Check if the PR title contains an exclamation mark (breaking)
+          if echo "$PR_TITLE" | grep -q '!'; then
+            # If the PR is breaking, ensure that the changeset contains minor changes
+            if ! echo "$AFFECTED_PACKAGES" | grep -q 'minor'; then
```

Comment:
Nit but should we be checking for major as well (when that day comes).

Reply:
I wouldn't say check for `major` since it will be rare, but maybe we could validate it both ways and handle cases where the changeset has `minor` and the title doesn't have a `!`.

Reply:
@arboleya we already do that [here](https://github.com/FuelLabs/fuels-ts/pull/2987/files#diff-900c14db30cf12f70161bd32e69363c032130443d4f052508d970d7d2fbfa472R108).